test/user_app_logging_test.go

```diff
@@ -60,32 +55,13 @@ var _ = Describe("logging", Ordered, func() {
 			err := k8s.CreateNamespaceWithMetadataE(GinkgoT(), options, nsObject)
 			Expect(err).ToNot(HaveOccurred())
 
-			setIdentifier := "integration-test-app-ing-" + namespace + "-green"
-
-			helloVar := map[string]interface{}{
-				"namespace": namespace,
-				"host":      host,
-				"ingress_annotations": map[string]string{
-					"external-dns.alpha.kubernetes.io/aws-weight":     "\"100\"",
-					"external-dns.alpha.kubernetes.io/set-identifier": setIdentifier,
-				},
-			}
-
-			tpl, err := helpers.TemplateFile("./fixtures/helloworld-deployment-v1.yaml.tmpl", "helloworld-deployment-v1.yaml.tmpl", helloVar)
-			Expect(err).ToNot(HaveOccurred())
-
-			err = k8s.KubectlApplyFromStringE(GinkgoT(), options, tpl)
-			Expect(err).ToNot(HaveOccurred())
-
-			k8s.WaitUntilIngressAvailable(GinkgoT(), options, "integration-test-app-ing", 8, 20*time.Second)
-
```

Comment:
Not sure why this existed, we don't need an ingress to collect logs that are output to stdout from the job